data/joeboadiDev.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name" : "Joe Boadi,
+  "displayStatsPublic" : "true",
```

Comment:
remove the line currently it's not needed.
```suggestion

```

Reply:
Done.